### ISSUE TITLE
Telegram Bot API doesn't support secret chats!

### ISSUE THREAD

**Issue by tctfsl** [tctfsl/telegram-fls-bot#5]
Just found out that the API doesn't support end-to-end encryption which I consider it as a showstopper if it were to be used in production. Unless there's support for secret chat in the future, this bot is unlikely to go live.

**Comment by tctfsl** [comment]
Revisiting this, let me rephrase: it will be difficult to implement this bot in an environment where there's a strict IT security policy. Since there's no end-to-end encryption, there will be concern that the bot or the user might transmit sensitive data.

**Comment by aqlanhadi** [comment]
more research https://stackoverflow.com/questions/48150276/is-telegram-bot-communication-secured-via-end-to-end-encryption
